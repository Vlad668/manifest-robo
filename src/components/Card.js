import React from 'react';

// class App extends React.Component {
//   render() {
//     return (
//       <div className="App">
//         <header className="App-header">
//           <img src={logo} className="App-logo" alt="logo" />
//           <p>
//             Edit <code>src/App.js</code> and save to reload.
//           </p>
//           <a
//             className="App-link"
//             href="https://reactjs.org"
//             target="_blank"
//             rel="noopener noreferrer"
//           >
//             Learn React
//           </a>
//         </header>
//       </div>
//     );
//   }
// }

const Card = (props) =>{
  const {id, name, email} = props
  
  return (
    <React.Fragment>
    <section key='097' id='test' className='tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5'>
    <img key='8' alt='profile' src={`https://robohash.org/${id}?200x200`}/>
    <div key='40'>
      <h3 key='90'>{name}</h3>
      <span key='9'>{email}</span>
    </div>
    </section>
    </React.Fragment>
  );
}

export default Card;
