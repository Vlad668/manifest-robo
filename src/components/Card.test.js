// import React from 'react'
// import { shallow,configure } from 'enzyme';
import Card from './Card'
// import '@testing-library/jest-dom';
// import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import { create } from "react-test-renderer"
import {shallow} from 'enzyme';


// configure({ adapter: new Adapter() });

const app = create(<Card />);

it('name of test', () => {
    expect(shallow(<Card/>).length).toEqual(1)
})

it('snapwatch test', () => {
    expect(app.toJSON()).toMatchSnapshot()
})

