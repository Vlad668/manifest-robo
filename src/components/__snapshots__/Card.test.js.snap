// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapwatch test 1`] = `
<section
  className="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
  id="test"
>
  <img
    alt="profile"
    src="https://robohash.org/undefined?200x200"
  />
  <div>
    <h3 />
    <span />
  </div>
</section>
`;
